darshan-util/pydarshan/benchmarks/benchmarks/report.py

```diff
@@ -67,6 +67,7 @@ class RecordCollectionToDF:
     params = [["examples/example-logs/ior_hdf5_example.darshan",
                "examples/example-logs/dxt.darshan",
                "tests/input/sample-dxt-simple.darshan",
+               "tests/input/sample-badost.darshan",
```

Comment:
And how much longer does the benchmark take now? Otherwise, the reviewer will need to run it themselves to check.

Reply:
The output for `time asv run -b "time_record_collection_to_df"`, ran locally on my Mac:
```
asv run -b "time_record_collection_to_df"  267.04s user 67.57s system 167% cpu 3:20.34 total
```

Reply:
> how much longer

Reply:
On `pydarshan-devel`:
```
asv run -b "time_record_collection_to_df"  202.47s user 52.02s system 162% cpu 2:36.31 total
```

So ~44 seconds longer.